app/transformations/ETMPToFrontEnd/API1833.scala

```diff
@@ -232,11 +232,14 @@ private object API1833ReadsUtilities extends Transformer {
         pathEtmpUnAuthorisedPaymentDetailsUnAuthorisedPmtType1.readNullable[JsString].map {
           case Some(paymentNature) =>
             dynamicPathFreeTxtIndividual(paymentNature.value).json.copyFrom(pathEtmpUnAuthorisedPaymentDetailsFreeTxtOrSchemeOrRecipientName.json.pick).orElse(doNothing)
+          case _ => throw new RuntimeException("Unknown EventOneReport behaviour")
```

Comment:
I think we need a Reads.failed here rather than throwing exception. Below too

Reply:
Done